README.md

```diff
@@ -1,28 +1,6 @@
-# python-lambda-template
+# SAP Invoices Flask App
 
-A template repository for creating Python lambda functions.
-
-## Repo Setup (delete this section and above after initial function setup)
-
-1. Rename "my_function" to the desired initial function name across the repo. (May be helpful to do a project-wide find-and-replace).
-2. Update Python version if needed (note: AWS lambda cannot currently support versions higher than 3.9).
-3. Install all dependencies with `make install`  to create initial Pipfile.lock with latest dependency versions.
-4. Add initial function description to README and update initial required ENV variable documentation as needed.
-5. Update license if needed (check app-specific dependencies for licensing terms).
-6. Check Github repository settings:
-   - Confirm repo branch protection settings are correct (see [dev docs](https://mitlibraries.github.io/guides/basics/github.html) for details)
-   - Confirm that all of the following are enabled in the repo's code security and analysis settings:
-      - Dependabot alerts
-      - Dependabot security updates
-      - Secret scanning
-7. Create a Sentry project for the app if needed (we want this for most apps):
-   - Send initial exceptions to Sentry project for dev, stage, and prod environments to create them.
-   - Create an alert for the prod environment only, with notifications sent to the appropriate team(s).
-   - If *not* using Sentry, delete Sentry configuration from my_function.py and test_my_function_.py, and remove sentry_sdk from project dependencies.
-
-# my_function
-
-Description of the function/functions.
+A Flask application run in AWS Lambda used to process invoices from Alma. Invoice processing is achieved by executing the ECS task that serves the [Alma SAP Invoices CLI](https://github.com/MITLibraries/alma-sapinvoices). The purpose of this web app is to provide a simple platform through which users can easily execute and monitor invoice processing. 
```

Comment:
I might propose "web accessible, graphical interface" instead of "platform" to describe what this Flask app accomplishes.

Reply:
Updated. :)